lib/cudnn/util.jl

```diff
@@ -26,3 +36,14 @@ scalingParameter(T, val) = error("Unknown tensor type $T")
 scalingParameter(::Type{Float16}, val) = Ref{Float32}(val)
 scalingParameter(::Type{Float32}, val) = Ref{Float32}(val)
 scalingParameter(::Type{Float64}, val) = Ref{Float64}(val)
+
+
+# Convert nothing to CU_NULL or C_NULL for ccall:
+cu_null(x) = (x === nothing ? CU_NULL : x)
```

Comment:
I'd use `something(x, CU_NULL)` instead of this function (and similar for `c_null`).